subscription-engine/kafka-ws/config/default.conf

```diff
@@ -0,0 +1,38 @@
+# LOGGER
+logger.transports.console.level=info
+logger.verbose:boolean=false
+
+logger.file.enable:boolean=false
+logger.file.level=info
+logger.file.filename=kafka-ws-logs-%DATE%.log
+
+# TICKET
+ticket.secret=0ed6004202bdd3ef04fb
+ticket.expiration.sec:integer=60
+
+# REDIS
+redis.host=kafka-ws-redis
+redis.port:integer=6379
+redis.db:integer=1
+
+# KAFKA CONSUMER
+consumer.group.id=kafka-ws
+consumer.metadata.broker.list=kafka:9092
+consumer.auto.offset.reset=largest
+
+# SERVER
+server.host=0.0.0.0
+server.port:integer=8080
+server.tls:boolean=false
+server.ca=/opt/kafka-ws/certs/ca-cert.pem
+server.key=/opt/kafka-ws/certs/server-key.pem
+server.cert=/opt/kafka-ws/certs/server-cert.pem
+server.jwt_exp_time:boolean=false
+server.connection_max_life_time:integer=7200
+server.request_cert:boolean:false
```

Comment:
Underscores should not be used as default, only in some special cases. For an example of special case, check the V2K config.